cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -961,9 +961,6 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size_t, 8)
     return;
   }
 
-  // if we have no work to do (eg, in a skip_rows/num_rows case) in this page.
-  if (s->num_rows == 0) { return; }
```

Comment:
For reviewers, this check is done more thoroughly here: https://github.com/rapidsai/cudf/blob/76f1c8ba9f2fd7ab6a6f3fd017ce11dd27963827/cpp/src/io/parquet/page_decode.cuh#L1145-L1161

Reply:
Ah, right.  I knew we were doing this kind of check, but forgot where we did it.  `is_page_contained` and `is_bounds_page` bring up bad memories :)